### ISSUE TITLE
Migrate to `WifiWizard` GitHub organization

### ISSUE THREAD

**Issue by parsonsmatt** [parsonsmatt/WifiWizard#63]
I'd like to migrate all of the `WifiWizard` related repositories off of my account, since I'm not doing anything with them anymore and can't provide adequate maintenance. It's probably best to have them on a github organization rather than an individual account.

If someone would like to create the organization, I can assign the repositories to that organization. I can also create the organization and do the assignments myself, and then assign the organization ownership to the current maintainer if that's preferred.

**Comment by emcniece** [comment]
I will be working with this plugin closely over the next year or so, and as such will have the opportunity to test PRs and perform updates. I'm happy to accept maintainership for this duration.

Organization created at https://github.com/WifiWizard-Cordova and you should have an invite already!

**Comment by vinicius73** [comment]
Please continue the project

**Comment by emcniece** [comment]
I'd love to, but the maintainer has been unresponsive so far.

**Comment by lepermunna** [comment]
Paging @hoerresb or @parsonsmatt ?
It might be best to re-create the project in a different repo if we don't hear from them

**Comment by emcniece** [comment]
Good idea. I am starting a dev cycle for this next week, perhaps it's a good time to port issues and PRs.

Anyone else interested in maintaining?
